packages/dso-toolkit/reference/render/highlight-box--link-list.html

```diff
@@ -6,6 +6,7 @@
       </li>
       <li><a href="#" class="download">Download link</a></li>
       <li><a href="#">Interne link</a></li>
+      <li><a href="#" aria-current="page">Current link</a></li>
```

Comment:
aria-current alleen als het in een nav staat, lijkt me?

Reply:
Ja, goed punt. Ik heb hem nu in het link-list voorbeeld weggehaald. Hij staat nog wel op de voorbeeldpagina waar deze lijst in een nav staat.